velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -567,13 +563,26 @@ ExpressionFuzzer::ExpressionFuzzer(
         continue;
       }
 
-      // Determine a list of concrete argument types that can bind to the
-      // signature. For non-parameterized signatures, these argument types will
-      // be used to create a callable signature. For parameterized signatures,
-      // these argument types are only used to fetch the function instance to
-      // get their determinism.
-      std::vector<TypePtr> argTypes;
-      if (signature->variables().empty()) {
+      if (!isDeterministic(function.first)) {
```

Comment:
No need to change anything in this PR, but I wonder if we can move this check into filterSignatures now that the check only needs function name.

Reply:
I see. Will follow up this change in a separate PR. Thanks.